src/test/components/ProfileViewer.test.js

```diff
@@ -6,7 +6,7 @@
 
 import * as React from 'react';
 import ReactDOM from 'react-dom';
-import { render } from '@testing-library/react';
+import { render } from 'firefox-profiler/test/fixtures/testing-library';
```

Comment:
nit: Can you move this import to the second half of the imports? When we have these type of separations with one empty line, it generally means that the first part is for imports from other external libraries like `@testing-library/react` and the second part is for our own imports. In this case, we are changing the external library import into our own project import.

I also see other imports in this PR with the same issue. I'm not writing a comment for all of them but it would be good to fix them as well.

Reply:
I orgranized all those imports. Please have a look and let me know.